NineDoF[] ninedofs
float32 delta_t